server/store/store.go

```diff
@@ -53,9 +53,12 @@ type Store interface {
 	TeamsToMattermostUserID(userID string) (string, error)
 	MattermostToTeamsUserID(userID string) (string, error)
 	CheckEnabledTeamByTeamID(teamID string) bool
-	ListGlobalSubscriptionsToCheck() ([]storemodels.GlobalSubscription, error)
+	ListGlobalSubscriptions() ([]storemodels.GlobalSubscription, error)
```

Comment:
```suggestion
	ListGlobalSubscriptions() ([]*storemodels.GlobalSubscription, error)
```
Why are we not using pointers in all these functions?